tests/SimpleStringTest.cpp

```diff
@@ -405,7 +411,7 @@ TEST(SimpleString, findNormal)
     SimpleString str("Hello World");
     LONGS_EQUAL(0, str.find('H'));
     LONGS_EQUAL(1, str.find('e'));
-    LONGS_EQUAL(-1, str.find('!'));
+    LONGS_EQUAL(SimpleString::npos, str.find('!'));
```

Comment:
Ok, we can keep the npos, although I find npos a horrible name (consistent with Standard C++ library chosing horrible names). But I would like the additional method as IMHO it is just clearer in the interface to have the additional method.

Using the size_t consistently was a good argument.